_rules/SVG-image-has-name.md

```diff
@@ -0,0 +1,141 @@
+---
+name: SVG image has accessible name
+description: |
+  This rule checks that each SVG image element that is included in the accessibility tree, has an accessible name.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any [SVG](https://www.w3.org/2000/svg) element with the [semantic role](#semantic-role) of `img` or `graphics-document`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element has a an accessible name.
```

Comment:
Should we add a note about decorative images? Something like:
SVG images used in a decorative role should be left out of the accessibility tree by using, e.g., `aria-hidden="true"`

Reply:
@carlosapaduarte SVG images are decorative by default. Are you saying that all SVG elements should have aria-hidden on them unless they have an accessible name because AT doesn't consistently follow the spec?

Reply:
Isn't it the same reason why we use img:alt="" in all decorative img elements? I think Brynanders is right and also img by default should be considered decorative.

Reply:
@Brynanders you're right. Please ignore my comment!